# A compressed array of path messages for a robot to follow, with goal at end
int16[] path
std_msgs/Float32 cost
